spec.md

```diff
@@ -15,4 +16,574 @@ The key words "unspecified", "undefined", and "implementation-defined" are to be
 An implementation is not compliant if it fails to satisfy one or more of the MUST, REQUIRED, or SHALL requirements for the protocols it implements.
 An implementation is compliant if it satisfies all the MUST, REQUIRED, and SHALL requirements for the protocols it implements.
 
+## Terminology
+
+| Term              | Definition                                       |
+|-------------------|--------------------------------------------------|
+| Bucket            | A flat organization that contains immutable objects.                          |
+| Brownfield Bucket | An existing back-end bucket.                        |
+| Greenfield Bucket | A new back-end bucket created by COSI.                        |
+|COSI  System              | Container Orchestration system/ Container Object Storage Implementation, communicates with Plugins using COSI service RPCs.                                     |
+| SP                | Object Storage Provider, the vendor of a COSI plugin implementation.                                                          |
+| CO                | Container Orchestrator, like Kubernetes.                                                          |
+| RPC/gRPC               | [Remote Procedure Call](https://en.wikipedia.org/wiki/Remote_procedure_call).                                         |
+| Node              | A host where the user workload will be running, uniquely identifiable from the perspective of a Plugin by a node ID. |
+| Driver(aka Plugin)           | Aka “plugin implementation”, a gRPC endpoint that implements the COSI Services.                                        |
```

Comment:
Should this be called "COSI Driver" to differentiate it from the CSI driver below?

Reply:
Fixed.